### DISCUSSION TITLE
Plot: Change spacing between Y-axis label and axis?

### DISCUSSION THREAD

**Discussion by metasim** [emilk/egui#4502] [Q&A] [answered]
How do I reduce the amount of space between the Y-axis label and the Y-axis itself?

<img width="242" alt="image" src="https://github.com/emilk/egui/assets/131013/0e9aad63-1161-431f-8b1a-1cbf18429e51">

Compared the the X-axis, the default spacing seems inordinately large:

<img width="266" alt="image" src="https://github.com/emilk/egui/assets/131013/887e2ca7-1182-4f19-aebc-c1e002cc283f">

I've tried playing with `AxisHints`, but the spacing doesn't seen affected by any of those settings.

**Comment by YgorSouza** [comment, marked as answer]
This distance is controlled by the [max_digits](https://docs.rs/egui_plot/0.27.2/egui_plot/struct.AxisHints.html#method.max_digits) method.
